hicommonwealth/commonwealth
# Fixed edit topics

## Link to Issue
Closes: #5837

## Description of Changes
- Adds nullable field to zod schema for edit topics

## Test Plan
- Go to discussions page
- Click on topics dropdown filter
- edit one topic
- after editing press save, make sure it works
- Refresh page, make sure topic stays updated (updated correctly)
